spec/fixtures.cr

```diff
@@ -0,0 +1,66 @@
+require "../src/config"
+require "../src/ext/scrypt_password"
+
+module DppmRestApi
+  struct Config
+    def initialize(@groups, @users)
```

Comment:
Why? If this initializer is needed, other may need it too – so it can be added to `/src/config.cr`.